osh/hub/osh_xmlrpc/worker.py

```diff
@@ -53,8 +56,15 @@
 
 @validate_worker
 def email_task_notification(request, task_id):
-    logger.debug('email_task_notification for %s', task_id)
-    return send_task_notification(request, task_id)
+    try:
+        logger.debug('email_task_notification for %s', task_id)
+        return send_task_notification(request, task_id)
+    finally:
+        if settings.ENABLE_SINGLE_USE_WORKERS:
```

Comment:
TODO: Same check needs to be added to the `email_scan_notification` function.

Reply:
Unless you are going to resolve this TODO before merging, please insert a TODO comment at the place in the code where the check needs to be added.

Reply:
I tried to fix it now.